mpt/impl/imap-mailbox/external-james/src/test/java/org/apache/james/mpt/imapmailbox/external/james/ExternalJamesTest.java

```diff
@@ -21,14 +21,16 @@
 
 import org.apache.james.mpt.imapmailbox.suite.DeploymentValidation;
 
+import org.apache.james.mpt.imapmailbox.suite.UidSearchOnLuceneIndex;
 import org.apache.james.mpt.onami.test.OnamiSuite;
 import org.apache.james.mpt.onami.test.annotation.GuiceModules;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 
 @RunWith(OnamiSuite.class)
 @Suite.SuiteClasses({
-    DeploymentValidation.class
+    DeploymentValidation.class,
+    UidSearchOnLuceneIndex.class
```

Comment:
I don't think we want to test indexing to validate a deployment, we'd rather have this in a mpt project like we do for ElasticSearch.